wiki/Shortcut_Key_Reference/en.md

```diff
@@ -1,179 +1,208 @@
-
--   **H**: Help (Contains all the shortcuts shown below)
-
-**Music**
-
--   **Z**: Stop
--   **X**: Play from start
--   **C or Spacebar**: Pause/Resume
--   **Left and Right arrow keys**: Skip through the song. Hold shift for 4x speed
-
-**Bookmarks commands**
-
--   **Ctrl+B**: Add bookmark at current location.
--   **Ctrl+Shift+B**: Remove bookmark at current location.
--   **Ctrl+Left Arrow**: Jump to previous bookmark.
--   **Ctrl+Right Arrow**: Jump to next bookmark.
-
-**Save and Reset**
-
--   **Ctrl+L**: Revert to saved
--   **Ctrl+S**: Quick save
--   **Ctrl+N**: Remove all notes (but keep timing/song setup)
-
-**Mode switch**
-
--   **F1**: Compose mode
--   **F2**: Design mode (only basics available at the moment)
--   **F3**: Timing mode
--   **F4**: Song setup
-
-#### Compose mode
-
--   **Left and Right arrow keys**: Move selected notes by 1 grid snap. Hold shift for 1 coordinate.
--   **Number keys 1-4**: Change selection/placement mode
--   **Q,W,E,R,L**: Change right-hand option modes
--   **Delete**: Remove selected notes
--   **Ctrl+Z,Y**: Undo,Redo
--   **Ctrl+A**: Select all notes
--   **Ctrl+X,C,V**: Cut, copy, paste
--   **Ctrl+D**: Clone the selection
--   **Alt while placing**: Snap distance based on time duration since last note
--   **Shift while placing**: No grid snapping
--   **Shift while modifying**: No grid or beat (time) snapping
--   **J/K**: Nudge circle backwards/forwards in time
--   **G**: Cycle through grid precision
--   **Ctrl+1/2/3/4**: Grid sizes (tiny,small,medium,large)
--   **Ctrl+Shift+R**: Rotate existing note by a specified angle
--   **Ctrl+ , (comma)**: Rotate Anticlockwise/Clockwise
--   **Ctrl+H,J**: Flip Horizontally/Vertically
-
-#### Timing mode
-
--   **T**: Tapping using the keyboard
--   **Shift while adjusting BPM/Offset**: 4x speed
--   **Ctrl+B**: Reset current timing point
--   **Ctrl+P**: Add timing point
+# Shortcut Key Reference
+
+There are many shortcuts in _osu!_ that can be utilized while in-game or while idle at any screen.
+They are listed out with their functions below.
+
+**Note:** some of these keys can be changed in the [Options](/wiki/Options/)!
+However, the list below presents the default keys.
+
+## Game-Wide
+
+These shortcuts work anywhere:
+
+- `Arrow Down`: Decrease volume.
+- `Arrow Up`: Increase volume.
+- `Alt` + `F4`: If playing/editing a beatmap, exit to song select; otherwise, this will close the osu!client.
+- `Ctrl` + `Alt` + `Shift` + `S`: Reload current skin (If playing a beatmap, this will queue the command to be executed after returning to song selection)
+- `Ctrl` + `F11`: Toggle frame times
+- `Ctrl` + `O`: Open options menu (does not work while playing in-game)
+- `F7`: Toggle frame limiter
+- `F8`: Toggle chat window
+- `F9`: Toggle extended chat window
+- `F10`: Toggle all mouse buttons (except for mouse wheel)
+- `F12`: Take a screenshot (hold `Shift` to upload instantly to the server for sharing.)
```

Comment:
this suggestion doesn't make anysense... I assume you meant:
``(while holding shift to upload instantly to the server for sharing.)``